crates/kernel/testsuite/moot_suite.rs

```diff
@@ -50,28 +68,29 @@ impl MootState {
         self,
         new_line_no: usize,
         line: &str,
-        db: Arc<dyn WorldStateSource>,
+        db: Arc<dyn Database + Send + Sync>,
     ) -> eyre::Result<Self> {
         let line = line.trim_end_matches('\n');
         match self {
             MootState::Ready {
                 ref session,
                 player,
             } => {
-                if let Some(rest) = line.strip_prefix(';') {
+                if line.starts_with([';', '%']) {
                     Ok(MootState::ReadingCommand {
                         session: session.clone(),
                         player,
                         line_no: new_line_no,
-                        command: rest.trim_start().to_string(),
+                        command: line[1..].trim_start().to_string(),
+                        command_kind: line.chars().next().unwrap().into(),
                     })
                 } else if let Some(new_player) = line.strip_prefix('@') {
                     Ok(MootState::new(session.clone(), Self::player(new_player)?))
                 } else if line.is_empty() || line.starts_with("//") {
                     Ok(self)
                 } else {
                     Err(eyre::eyre!(
-                        "Expected a command (starting `;`), a comment (starting `//`), a player switch (starting `@`), or an empty line"
+                        "Expected a command (starting `;`), a comment (starting `//`), a player switch (starting `@`), a command (starting `%`), or an empty line"
```

Comment:
Running out of prefixes lol. I tried `$` initially but that conflicts with `$object` as an expectation. `%` ought to work.